client/src/modules/email-report/index.html

```diff
@@ -0,0 +1,217 @@
+                  ng-messages="EmailReportRegistrationForm.email.$error"
+                  ng-show="EmailReportRegistrationForm.$submitted">
+                  <div ng-messages-include="modules/templates/messages.tmpl.html">
+                  </div>
+                </div>
+              </div>
+            </div>
+            <!-- frequency -->
+
+            <div
+              class="form-group"
+              ng-class="{'has-error' : EmailReportRegistrationForm.frequency.$invalid && EmailReportRegistrationForm.$submitted}">
+
+              <label class="control-label ng-binding" translate>
+                FORM.LABELS.FREQUENCY
+              </label>
+
+              <div class="input-group col-lg-12">
+
+                <select
+                  required=""
+                  name="frequency"
+                  ng-model="EmailReportCtrl.emailReport.frequency"
+                  class="form-control">
+                    <option  
+                      ng-repeat="item in EmailReportCtrl.frequencies"
+                      value="{{item}}">
+                        {{item}}
+                    </option>
+                </select>
+                 <div class="help-block"
+                  ng-messages="EmailReportRegistrationForm.frequency.$error"
+                  ng-show="EmailReportRegistrationForm.$submitted">
+                    <div ng-messages-include="modules/templates/messages.tmpl.html">
+                    </div>
+                </div>
+
+              </div>
+            </div>
+
+
+            <!-- report Group -->
+
+            <div
+              class="form-group"
+              ng-class="{'has-error' : EmailReportRegistrationForm.code_report_group.$invalid && EmailReportRegistrationForm.$submitted}">
+
+              <label class="control-label ng-binding" translate>
+                FORM.LABELS.REPORT_GROUP
+              </label>
+
+              <div class="input-group col-lg-12">
+                <select 
+                name='code_report_group'
+                  required=""
+                  ng-model="EmailReportCtrl.emailReport.code_report_group"
+                  class="form-control">
+                    <option 
+                      ng-repeat="item in EmailReportCtrl.report_groups"  
+                      value="{{item.code}}">
+                        {{item.name}}
+                    </option>
+                </select>
+                <div class="help-block"
+                  ng-messages="EmailReportRegistrationForm.code_report_group.$error"
+                  ng-show="EmailReportRegistrationForm.$submitted">
+                    <div ng-messages-include="modules/templates/messages.tmpl.html">
+                    </div>
+                </div>
+              </div>
+            </div>
+
+
+
+            <bh-loading-button 
+             ng-show="EmailReportCtrl.selectedEmailReport.selected==false"
+             class="pull-right" loading-state="BedRegistrationForm.$loading">
```

Comment:
Instead of having multiple `pull-right` classes, it would be best to wrap all buttons in an element that is `pull-right`.  Something like this:

```html
<div class="pull-right">
   <button></button>
   <button></button>
   <button></button>
</div>
```

That way the browser does less calculation when the layout changes.